### ISSUE TITLE
Technology list should be paginated

### ISSUE THREAD

**Issue by brunocarneiro** [ciandt-dev/tech-gallery#179]

**Comment by larruda** [comment]
I suggest using "infinite scroll" technique for this.

**Comment by brunocarneiro** [comment]
Yep. But when implementing we must think about how much data will be loaded.
Maybe if the user scrolls too much, page get slowly. Maybe infinite scroll loading next and previous data.
